### PR TITLE
aws quicksight: add page

### PR DESCRIPTION
**Pull request by ahonarmand**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/aws-quicksight.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# aws quicksight
+
+> CLI for AWS QuickSight - access QuickSight entities.
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:3]:
```suggestion
> CLI for AWS QuickSight.
> Access QuickSight entities.
```

File: pages/common/aws-quicksight.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/quicksight/index.html>.
+
+- Show data-sets:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:6]: Based on the AWS doc page
```suggestion
- List datasets:
```

File: pages/common/aws-quicksight.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Show groups:
+
+`aws quicksight list-groups --aws-account-id {{aws_account_id}} --namespace default`
+
+- Show dashboards:
+
+`aws quicksight list-dashboards --aws-account-id {{aws_account_id}}`
+
+- See details of a dataset such as its input and output columns:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:22]: Maybe this is better:
```suggestion
- Display detailed information about a dataset:
```

File: pages/common/aws-quicksight.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`aws quicksight describe-data-set --aws-account-id {{aws_account_id}} --data-set-id {{data_set_id}}`
+
+- See who has access to the dataset and what kind of actions they can perform on the dataset:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:26]:
```suggestion
- Display who has access to the dataset and what kind of actions they can perform on the dataset:
```

File: pages/common/aws-quicksight.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# aws quicksight
+
+> CLI for AWS QuickSight - access QuickSight entities.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/quicksight/index.html>.
+
+- Show data-sets:
+
+`aws quicksight list-data-sets --aws-account-id {{aws_account_id}}`
+
+- Show users:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:10]:
```suggestion
- List users:
```

File: pages/common/aws-quicksight.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`aws quicksight list-users --aws-account-id {{aws_account_id}} --namespace default`
+
+- Show groups:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:14]:
```suggestion
- List groups:
```

File: pages/common/aws-quicksight.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`aws quicksight list-groups --aws-account-id {{aws_account_id}} --namespace default`
+
+- Show dashboards:
```
Comment (navarroaxel) [inline pages/common/aws-quicksight.md:18]:
```suggestion
- List dashboards:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @ahonarmand, welcome and thanks for your submission. I left some comments

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @ahonarmand!

**Pull request merged**